opennmt/utils/exporters.py

```diff
@@ -108,8 +111,10 @@ def _export_model(self, model, export_dir):
             tf.lite.OpsSet.TFLITE_BUILTINS,  # enable TensorFlow Lite ops.
             tf.lite.OpsSet.SELECT_TF_OPS,  # enable TensorFlow ops.
         ]
-        converter.optimizations = [tf.lite.Optimize.DEFAULT]
-        if self._quantization == "float16":
+        if self._quantization == "dynamic_range":
+            converter.optimizations = [tf.lite.Optimize.DEFAULT]
+        elif self._quantization == "float16":
+            converter.optimizations = [tf.lite.Optimize.DEFAULT]
```

Comment:
Looks like you can keep the line `converter.optimizations = [tf.lite.Optimize.DEFAULT]` outside the `if` block and remove the first `if`?

Reply:
I think I have to have the 2 `if` statements.  I can't move the optimize line to outside the `if` statement or else it will quantize when exporting with the argument `tflite`.

Reply:
Yes, I realized that after my comment.